static/client.html

```diff
@@ -25,6 +31,19 @@
       font-family: sans-serif;
     }
 
+    @media (prefers-color-scheme: dark) {
```

Comment:
I think this can be replaced by 

```CSS
    @media (prefers-color-scheme: dark) {
      html { 
        background-color: #222222; 
        color: #dddddd; 
      }
    
     #banner {
        background-color: #226644;
      }
    
      .executing {
        background-color: #662233;
      }
    }
```
Also it might be better to move it down so it would override any other 'standard' style.
![image](https://user-images.githubusercontent.com/25481501/107485660-f7f35180-6b7b-11eb-8f50-be3bd45836a7.png)